### DISCUSSION TITLE
Welcome to wiperf Discussions!

### DISCUSSION THREAD

**Discussion by wifinigel** [wifinigel/wiperf#31] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

__Don't forget that the documentation for Wiperf can be found at http://wiperf.net. Please check here before asking questions about installing and using wiperf.__

Please feel free to add your ideas and experiences of wiperf and don't forget to add a relevant category to your contribution when opening a new discussion. Thanks!

**Comment by wudave** [comment]
Thanks for the comprehensive docs.  I installed v2.0.1 image and found an inconsistent documentation in https://wifinigel.github.io/wiperf/grafana_configure/#adding-wiperf-dashboards

On my device, the "dashboards" folder is under /etc/wiperf/dashboards rather than /usr/share/wiperf/dashboards.
wlanpi@wlanpi:/etc/wiperf$ ls -la
total 48
drwxr-xr-x   4 root root  4096 May  6 22:11 .
drwxr-xr-x 109 root root  4096 May  7 00:58 ..
drwxr-xr-x   3 root root  4096 Jan 27 09:46 conf
-rw-r--r--   1 root root 13956 Jan 27 09:18 config.default.ini
-rw-r--r--   1 root root 14058 May  6 22:11 config.ini
drwxr-xr-x   4 root root  4096 Jan 27 09:46 dashboards
